### DISCUSSION TITLE
记录一个我遇到的考研炮灰 | YuZhangWang的领域

### DISCUSSION THREAD

**Discussion by giscus** [YuZhangWang/YuZhangWang.github.io#14] [Announcements]
# 记录一个我遇到的考研炮灰 | YuZhangWang的领域

我遇到了一个自信满满，不知悔改的考研炮灰，这是我从被他各种无脑行为打扰，到了解到他学习情况，最后物理心灵上共同逃离的故事。 1. 和学习无关的内容 1.1 长时间的噪音 基本情况 这人坐我后面，每天餐巾纸都要用掉两包，一天到晚抽餐巾纸，抽纸声音一来很频繁，二来他就好像和餐巾纸有仇一样，抽的时候，巨用力，这样每次声音都响，每天抽那么多下，已经形成噪音了。 第1次沟通失败 后来，晚上教室就剩我和他了，

https://yuzhang.wang/122-cannon-fodder/

**Comment by yidimopozhibai** [comment]
老哥考哪里啦

**Reply by YuZhangWang** [reply]
@SEU

**Comment by knight965** [comment]
一战杭师失败，我可能要换个学校，，，怎么看学校计算机专业强不强？我感觉所谓的学科评级水分很多，，，

**Reply by YuZhangWang** [reply]
简单来说，比你的本科好就行。再就是看你的追求，也就是你想考什么学校，决定之后干就完事了。

**Reply by knight965** [reply]
确实，纠结择校不如多写两道题(´-_ゝ-`)

**Comment by hjljy** [comment]
祝早日上岸

**Reply by YuZhangWang** [reply]
感谢🙏🙏🙏

**Comment by xiaoxiudog** [comment]
博主考的是生物硕士吗
